queue_job/models/queue_job.py

```diff
@@ -338,6 +343,9 @@ class JobChannel(models.Model):
     job_function_ids = fields.One2many(comodel_name='queue.job.function',
                                        inverse_name='channel_id',
                                        string='Job Functions')
+    removal_interval = fields.Integer(
```

Comment:
It is a personal choice but I would have chosen the name "time_to_live" instead of removal_interval.

Reply:
I just kept the original name